dissect/target/plugins/child/hyperv.py

```diff
@@ -40,19 +40,21 @@ def check_compatible(self):
             raise UnsupportedPluginError("No registered VMs and no data.vmcx file found")
 
     def list_children(self) -> Iterator[ChildTargetRecord]:
+        for xml_path in self.vm_xml:
+            yield ChildTargetRecord(
+                type=self.__type__,
+                path=xml_path.resolve(),
+                _target=self.target,
+            )
+
         if self.data_vmcx.exists():
             data = hyperv.HyperVFile(self.data_vmcx.open()).as_dict()
+            if "VirtualMachines" not in data["Configurations"]:
+                return
 
             for vm_path in data["Configurations"]["VirtualMachines"].values():
                 yield ChildTargetRecord(
                     type=self.__type__,
                     path=path.from_windows(vm_path),
                     _target=self.target,
                 )
```

Comment:
```suggestion
            if virtual_machines := data["Configurations"].get("VirtualMachines"):
                for vm_path in virtual_machines.values():
                    yield ChildTargetRecord(
                        type=self.__type__,
                        path=path.from_windows(vm_path),
                        _target=self.target,
                    )
```
Or something along those lines, then you can keep the other code where it was.